app/assets/stylesheets/application.scss

```diff
@@ -390,6 +390,10 @@ $phases: (beta, #f47738),
   margin: 0;
 }
 
+.form-hint {
+  width: 50%;
```

Comment:
All form elements have a max width of 50% so they go inside a `column-one-whole` which means the form-hint extends beyond the inputs(because theres a lot of text) so this is my work around to ensure they are a consistent width.